.github/scripts/start_dd_agent.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+DD_API_KEY=$1
+dataset=$2
+config=$3
+
+echo "Dataset: ${dataset}"
+echo "Config: ${config}"
+
+# Install Datadog system agent
+DD_AGENT_MAJOR_VERSION=7 DD_API_KEY=$DD_API_KEY DD_SITE="datadoghq.eu" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+DATADOG_YAML_PATH=/etc/datadog-agent/datadog.yaml
+sudo chmod 666 $DATADOG_YAML_PATH
+
+# Associate metrics with tags and env
+{
+    echo "env: rasa-regression-tests"
+    echo "tags:"
```

Comment:
I think it'd be useful to add a PR number, git sha, and branch name as tags.

Reply:
You can try with `${{ github.event.number }}` for getting the PR ID.

Reply:
Thanks for the suggestion. I will keep this in mind for the future, but at the moment I am not sure about all the implications:
The challenge with the PR number is to make ensure this also works when the MR test haven't been triggered by a PR, but some other event (push, or schedule). The datadog tag can't be empty, so maybe in this case we need a default value.

Reply:
Good catch. In that scenario, we could try to assign "schedule" as default when `$PR_ID` is unset or empty. Could that work?

```
    echo "- pr_id:${PR_ID:-schedule}"
```

Reply:
That should work, I added the PR_ID tag